win32ss/user/ntuser/ntstubs.c

```diff
@@ -513,10 +515,7 @@ NtUserCreateInputContext(ULONG_PTR dwClientImcData)
 
```

Comment:
NB: `dwClientImcData` check could be moved before `UserEnterExclusive()`.

Reply:
Moved.